sync.cmd

```diff
@@ -0,0 +1,25 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set _args=%*
+if /I [%1]==[/?] goto Usage
+if [%1]==[]  set _args=-p
+
+echo Running executor.cmd %_args%
+call executor.cmd %_args%
```

Comment:
I've still not had the chance to look through all your changes but I do prefer the later (i.e. different options to `executor.cmd sync` as opposed to different executor commands.